ms-docs/JBoss EAP on Azure.MD

```diff
@@ -1,161 +1,164 @@
 ---
-title: Quickstart - JBoss EAP on Azure for enterprise Java applications
-description: Deploy enterprise Java apps using JBoss EAP with Red Hat Enterprise Linux (RHEL) VM on Azure
-author: SpektraSystems
-ms.author: Theresa Nguyen
+title: Quickstart - JBoss EAP on Azure RHEL VM and VMSS
+description: How to deploy enterprise Java applications using Red Hat JBoss EAP on an Azure RHEL VM and VMSS.
+author: theresa-nguyen
+ms.author: bicnguy
 ms.topic: quickstart
-ms.date: 07/30/2020
+ms.service: virtual-machines-linux
+ms.subservice: workloads
+ms.assetid: 8a4df7bf-be49-4198-800e-db381cda98f5
+ms.date: 08/17/2020
 ---
 
 # Deploy enterprise Java applications to Azure with JBoss EAP on Red Hat Enterprise Linux
 
-This quickstart shows you how to deploy [JBoss Enterprise (EAP)](https://www.redhat.com/en/technologies/jboss-middleware/application-platform) on [Red Hat Enterprise Linux (RHEL)](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) to Azure Virtual Machines (VM) and Virtual Machine Scale Sets (VMSS). JBoss EAP is an open source application server platform that delivers enterprise-grade security, scalability, and performance. RHEL is an open source operating system (OS) platform, where you can scale existing apps and roll out emerging technologies across all environments. JBoss EAP and RHEL includes everything needed to build, run, deploy, and manage enterprise Java applications in any environment, including on-premises, virtual environments, and in private, public, or hybrid clouds.
+These Quickstarts templates will show you how to deploy [JBoss Enterprise (EAP)](https://www.redhat.com/en/technologies/jboss-middleware/application-platform) on [Red Hat Enterprise Linux (RHEL)](https://www.redhat.com/technologies/linux-platforms/enterprise-linux) to Azure Virtual Machines (VM) and Virtual Machine Scale Sets (VMSS), run a sample Java app and validate your deployment. JBoss EAP is an open source application server platform that delivers enterprise-grade security, scalability, and performance. RHEL is an open source operating system (OS) platform, where you can scale existing apps and roll out emerging technologies across all environments. JBoss EAP and RHEL includes everything needed to build, run, deploy, and manage enterprise Java applications in any environment, including on-premises, virtual environments, and in private, public, or hybrid clouds.

```

Comment:
The Link mentioned for Red Hat Enterprise Linux (RHEL) is not accessible without en/
https://www.redhat.com/technologies/linux-platforms/enterprise-linux